components/ApplicationTable.tsx

```diff
@@ -149,6 +150,7 @@ const ApplicationTable: FC<ApplicationTableProps> = ({
             Logged in as: <strong>{email}</strong>
           </Text>
         </Card>
+        <HomepageLinkButton />
```

Comment:
Could we keep the `HomepageLinkButton` and the `Logged in as` element both on the rhs of their outer container? Atm the `Logged in as` element floats in the center (because of `justify="between"` in the outer `Flex`)